inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndex.java

```diff
@@ -477,11 +477,12 @@ public void indexDocument(AnnotationDocument aDocument, JCas aJCas) throws IOExc
     private void deindexDocument(long aSourceDocumentId, long aAnnotationDocumentId, String aUser)
         throws IOException
     {
-        indexWriter.deleteDocuments(new Term(FIELD_ID,
-                String.valueOf(aSourceDocumentId) + "/" + String.valueOf(aAnnotationDocumentId)));
-
-        indexWriter.commit();
+        if (indexWriter != null) {
+            indexWriter.deleteDocuments(new Term(FIELD_ID, String.valueOf(aSourceDocumentId) + "/"
+                    + String.valueOf(aAnnotationDocumentId)));
```

Comment:
Why do you need `String.valueOf`? If you add a long and a String, then it should result in a String.

Reply:
OK.